src/main/java/jalasoft/com/models/ToDoListPersister.java

```diff
@@ -0,0 +1,53 @@
+package jalasoft.com.models;
+
+import java.util.List;
+import java.util.UUID;
+
+public class ToDoListPersister {
+
+	private static Page<Task> page = new Page<Task>(10);
+	private static Task editingTask;
+
+	public static void addTask(Task task) {
+		page.addItem(task);
+	}
+
+	public static void removeTask(Task task) {
+		page.removeItem(task);
+	}
+
+	public static void removeTask(UUID taskId) {
+		for (int i = 0; i < page.getItems().size(); i++) {
+			if (taskId.equals(page.getItems().get(i).getId())) {
+				removeTask(page.getItems().get(i));
+			}
+		}
+	}
+
+	public static void updateTask(Task task) {
+		removeTask(editingTask);
+		addTask(task);

```

Comment:
why when updating a task you have to remove the editingTask? is the editingtask same than the task?

Reply:
Hi, "editingTask" is where the old attributes are stored and "task" is where the new ones are. I am first removing the old task and then adding the modified one. They are not the same. Maybe that is not the best way to "update" right?